CoreNotesView/src/au/gov/asd/tac/constellation/views/notes/NotesViewController.java

```diff
@@ -44,6 +51,7 @@ public class NotesViewController {
 
     private static final String NOTES_READ_STATE = "Notes View: Read State";
     private static final String NOTES_WRITE_STATE = "Notes View: Write State";
+    private static final Logger LOGGER = Logger.getLogger(NotesViewController.class.getName());
```

Comment:
seems like the logger isn't being used? probably means there is a number of unused imports